test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1685,6 +1696,213 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
         }
       });
     });
+
+    context('Case 5: not impemented', metadata, function () {}).skipReason = 'not implemented';
+
```

Comment:
```suggestion
    context('Case 5: not implemented', metadata, function () {}).skipReason = 'not implemented';
```

But should we leave the case name for reference? "`tlsDisableOCSPEndpointCheck` is permitted"